GrandCentralBoard/Core/Configuration/WidgetSettings.swift

```diff
@@ -68,8 +68,18 @@ extension WidgetSettings : Hashable {
     }
 }
 
-public func == (lhc: WidgetSettings, rhc: WidgetSettings) -> Bool {
-    return lhc.hashValue == rhc.hashValue
+public func == (lhs: WidgetSettings, rhs: WidgetSettings) -> Bool {
```

Comment:
Looks good to me. I can't find any better solution. But I'm thinking if lines 37-68 are needed, because `hashValue` will never be used.